__tests__/app.test.js

```diff
@@ -185,6 +185,7 @@ describe("app", () => {
                 topic: expect.any(String),
                 created_at: expect.any(String),
                 votes: expect.any(Number),
+                comment_count: expect.any(String),
```

Comment:
don't amend previous tests to check new behaviour, you should be adding a new test!